api/types/src/transaction.rs

```diff
@@ -68,11 +68,20 @@ pub enum TransactionData {
     OnChain(TransactionOnChainData),
     /// A transaction currently sitting in mempool
     Pending(Box<SignedTransaction>),
+    /// A transaction committed to storage but not yet to internal indexer yet.
+    PendingQueryable(TransactionOnChainData),
```

Comment:
Change of protocol is not ideal.. let's just pretend those are in mempool.. :trollface: